lms/services/oauth_http.py

```diff
@@ -96,7 +96,7 @@ def get_access_token(self, token_url, redirect_uri, auth, authorization_code):
         )
 
     def refresh_access_token(
-        self, token_url, redirect_uri, auth, prevent_concurrent_refreshes=False
+        self, token_url, redirect_uri, auth, prevent_concurrent_refreshes=True
```

Comment:
The flag has been left in so it can easily be reverted if necessary. We can remove at some later point.

Reply:
I'd say just remove it, it'll still be easy to put it back. Otherwise you'll likely forget to ever remove it in future.